aviary/models/large_single_aisle_1/large_single_aisle_1_GwGm.csv

```diff
@@ -156,14 +156,14 @@ mission:taxi:duration,0.1677,h
 settings:equations_of_motion,2DOF
 settings:mass_method,GASP
 
-# Initial Guesses
+# Initialization Guesses
```

Comment:
I think "initial guesses" sounds better than "initialization guesses". (and is close to the math term for PDEs "initial conditions")

What the motivation for the change?

Reply:
the intitial_guesses in phase_info are the initial conditions used throughout the trajectory for the optimizer.
These are variables used to initialize the values of other variables.

I wanted to disambiguate the two of them